app/lib/backend/storage.rb

```diff
@@ -7,8 +7,7 @@ def self.parse_config
       config = File.read("#{Rails.root}/config/backend_redis.yml")
       config = ERB.new(config).result(binding)
       config = YAML.load(config)
-      config = config.fetch(Rails.env).symbolize_keys
-      config.reverse_merge(logger: Rails.logger)
```

Comment:
This parameter it's causing problems with the Redis client and I don't think it's being used for anything.

Reply:
why remove `      config = config.fetch(Rails.env).symbolize_keys`?

Also did you check what happened with log in this case?

Reply:
that line is not removed, it's modified to `config.fetch(Rails.env).deep_symbolize_keys`

I didn't check those logs, just checked that nothing crashed. I assumed it would crash if it were actually being used from somewhere. Also I don't know exactly what should I look for in order to notice a difference in logs.

Reply:
wrt differences in log - does it use redis or sidekiq logger now? Or just prints on the stdout?

Reply:
The `:logger` param was accepted by previous versions of the `redis` gem (< 5). But it was removed for performance issues. So after the update, the gem doesn't log anything.